modules/mobile/docs/schemas/v2/User.yml

```diff
@@ -0,0 +1,40 @@
+type: object
+required:
+  - id
+  - type
+  - attributes
+properties:
+  id:
+    type: string
+    example: "abe3f152-90b0-45cb-8776-4958bad0e0ef"
+  type:
+    type: string
+    example: "user"
+  attributes:
+    type: object
+    required:
+      - id
+      - firstName
+      - middleName
+      - lastName
+      - signinEmail
+      - signinService
```

Comment:
isCernerPatient looks to be missing from required.

Reply:
@kpethtel is this something that you think should be part of the facilities endpoint instead of the user endpoint?

Reply:
Facilities info does already have cerner field and the front end uses it for `userHasCernerFacilities`. They use that method instead of isCernerPatient, afaict. I'll ask them if they think it's worth getting rid of, but it also doesn't seem to have any carrying cost.

Reply:
Yeah, as long as it doesn't require calling an additional upstream then it's whatever

Reply:
The user instantiation code is very difficult to follow, but I believe it likely does make a request to MPI for this data if that data isn't already cached. Given that this is the user endpoint and probably the first thing the mobile app fetches, it may not be cached. The code is so hard to follow that I can't be totally sure, especially given that the vets-api pre-loads some (but not all) data and caches some (but maybe not all) data. 

I've asked the FE team about this. If they say it's not necessary, we can remove it. But I also think we should generally be able to call most methods on the user object without concern.

Reply:
No one has answered my question, so I'm just going to say I think we can safely remove it. If front end wants it back, they can ask for it back when they implement the front end of this ticket. It does seem like accessing the info can make an upstream request so we should avoid calling it if we don't need to.

Reply:
In the specs, it may be handled by the factories. But I doubt this should be a big issue either way. It's also possible that the user gets fully loaded by the authentication process and it's just not clear from the code. Very hard to say.

Reply:
Removed!